src/SimpleList/manifest.json

```diff
@@ -530,8 +530,8 @@
           "default": "No List Items",
           "styles": {
             "fontFamily": "@heading",
-            "fontWeight": "600",
-            "color": "@text",
+            "fontWeight": "medium",
```

Comment:
I'm pretty sure this actually has to stay as a number. I'd be skeptical that `"medium"` would work. I'd double check with @zachbharris though

Reply:
It looks like you're correct, I'll switch that to a number